sdks/python/container/base_image_requirements_manual.txt

```diff
@@ -25,9 +25,7 @@
 # Consider constraining requirements of Beam itself when necessary.
 
 bs4  # Commonly used HTML processing tool.
-# Don't upgrade to Cython 3.x, until it's released, stable and we have consensus
-# to upgrade. Use 0.xx for now.
-cython<1
+cython==3.0.0
```

Comment:
I don't think we should pin a specific version here. This defines container dependencies. container dependencies don't change after a container image is released. Removing the pin means we will regularly update to the latest version.